### PR TITLE
[W5.10h][W13-A4]Yeo Guek Ling

### PR DESCRIPTION
**Pull request by guekling**

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -88,6 +88,32 @@ Use case ends.
...
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to find persons with specific tag
+2. AddressBook shows a list of persons with specific tag
+3. User requests to rename the specific tag
+4. AddressBook requests for confirmation
+5. AddressBook renames the specific tag and shows a list of persons with renamed tag <br>
```
Comment (brandonyeoxg) [inline doc/DeveloperGuide.md:99]: The Use Case seeks to model the interaction between the Actor and System. In such a way user -> system -> user -> system -> etc.
Can you spot the place to modify? =P

**Review by brandonyeoxg — COMMENTED** [review]
@guekling Some comments added. Please close the PR after reading comments. Good effort keep it up!

File: doc/DeveloperGuide.md (hunk 1 above)
Comment (guekling) [reply doc/DeveloperGuide.md:99]: Between Step 4 and Step 5, User should confirm the renaming of tag?

Comment (brandonyeoxg) [reply doc/DeveloperGuide.md:99]: That would indeed be a better flow of events right =) ?

**Pull request closed without merging**